regression-tests/runtests.sh

```diff
@@ -81,12 +81,8 @@ Check() {
     reffile=`echo $1 | sed 's/[.][^.]*$//'`
     basedir=`dirname $1`
 
-    # If the library isn't there, build it
-    if [ ! -f "${SSMLIBDIR}/libssm.a" ] ; then
-	Run mkdir -p out &&
-	Run make -C "$SSMDIR" build/libssm.a "1>&2"
-    fi
-    
+    Run make -C "$SSMDIR" build/libssm.a "1>&2"
```

Comment:
Please move this outside the Check() function (e.g., to right before Check() is called on the command-line arguments) for speed. There's no need to try to make it more than once.

Reply:
Done